projects/win32/tomviz.bundle.cmake

```diff
@@ -25,7 +25,7 @@ install(DIRECTORY "${install_location}/share/"
         COMPONENT ${AppName})
 
 # install paraview python modules and others.
-install(DIRECTORY "${install_location}/lib/paraview"
+install(DIRECTORY "${install_location}/lib"
```

Comment:
Below, `lib/tomviz` and `lib/itk` are installed. I'm not sure if this is redundant, since now the root `lib` directory is getting installed. Should those be removed?